packages/foam-vscode/package.json

```diff
@@ -48,6 +48,13 @@
     "configuration": {
       "title": "Foam",
       "properties": {
+        "foam.files.ignore": {
+          "type": [
+            "array"
+          ],
+          "default": [],
+          "description": "List of files/folders Foam should ignore. Paths are relative to the workspace root."
```

Comment:
this needs to be updated right? Might also be worth adding an example in parenthesis for clarity

Reply:
Instead of an example we can set a default `.vscode/*` and `_layouts/*`for example.
Description wise I'll try to make it clearer since the description is out of date